src/main/java/io/appium/java_client/MobileCommand.java

```diff
@@ -72,10 +72,7 @@ static CommandInfo postC(String url) {
         return new CommandInfo(url, HttpMethod.POST);
     }
 
-    private static Map<String, CommandInfo> getMobileCommands() {
-        if (commandRepository != null) {
-            return commandRepository;
-        }
+    protected static ImmutableMap.Builder<String, CommandInfo> getCommandsBuilder() {
```

Comment:
Ah that was because i was overriding/hiding this method in the IOSMobileCommand class. I will remove when i put this command in the MobileCommand class.